filebeat/autodiscover/builder/hints/logs_test.go

```diff
@@ -328,15 +328,15 @@ func TestGenerateHints(t *testing.T) {
 				},
 				"hints": common.MapStr{
 					"logs": common.MapStr{
-						"module":         "apache2",
+						"module":         "apache",
 						"fileset.stdout": "access",
 						"fileset.stderr": "error",
 					},
 				},
 			},
 			len: 1,
 			result: common.MapStr{
-				"module": "apache2",
+				"module": "apache",
```

Comment:
Was it failing? I left it intentionally after adding the option for moved modules.

Reply:
yes, but probably explained by the removal of the other file. I will revert this change.